contrib/awssqs/samples/awssqs-source.yaml

```diff
@@ -1,11 +1,10 @@
 # Replace the following before applying this file:
 #   QUEUE_URL: Replace with the AWS SQS queue.
-#   QUEUE_NAME: Replace with the name of the queue (to use as name)
 
 apiVersion: sources.eventing.knative.dev/v1alpha1
 kind: AwsSqsSource
 metadata:
-  name: QUEUE_NAME-source
+  name: awssqs-sample-source
```

Comment:
this seems wrong as per the directions with 'sed'

Reply:
Thx, fixing this and the credentials property below that is also no longer applicable (I think I messed up a rebase a few weeks ago)